src/App.js

```diff
@@ -2,15 +2,18 @@ import { Routes, Route } from 'react-router-dom';
 import Navbar from './components/Navbar';
 import Books from './components/books/Books';
 import Categories from './components/categories/Categories';
+import './scss/base.scss';
 
 function App() {
   return (
     <>
       <Navbar />
-      <Routes>
-        <Route exact path="/" element={<Books />} />
-        <Route path="/categories" element={<Categories />} />
-      </Routes>
+      <main className="site-content">
+        <Routes>
+          <Route exact path="/" element={<Books />} />
+          <Route path="/categories" element={<Categories />} />
+        </Routes>
+      </main>
```

Comment:
- [ ] Please make sure that all elements in design are added. Take a look at the screenshots below and kindly compare and fix differences such as  👇
- avatar icon on the navbar;
- book information section which is in the middle also has several differences;
- in the add new book form replace author with category;
- Original
![originale](https://user-images.githubusercontent.com/67087664/179000962-952773be-464a-4e2a-8002-386f47e11890.png)
- Your design with marked differences
![Screenshot (290)_LI](https://user-images.githubusercontent.com/67087664/179001234-47deb22d-d35d-4cb1-886e-3a5c2e0c8525.jpg)